numpy/core/tests/test_datetime.py

```diff
@@ -400,6 +398,38 @@ def test_timedelta_scalar_construction(self):
         assert_raises(TypeError, np.timedelta64, a, 'M')
         assert_raises(TypeError, np.timedelta64, a, 'Y')
 
+    @pytest.mark.parametrize("size", [
+        3, 21, 217, 1000])
+    def test_timedelta64_nat_argsort_stability(self, size):
```

Comment:
Because this test and the one below are effectively copy/paste with `M8` -> `m8` from the first PR, we could also consider parametrizing over those two dtypes to reduce duplication. Maybe I just get carried away with the parameters though :) It would take a little more work, so not sure on the tradeoff value for maintainability.

Reply:
Done! Looks lot cleaner now. Thanks!